packages/node-resizer/src/NodeResizer.vue

```diff
@@ -34,6 +34,7 @@ export default {
     :node-id="nodeId"
     :position="c"
     :variant="ResizeControlVariant.Line"
+    :aspect-ratio="props.aspectRatio"
```

Comment:
the `props` prefix can be removed as props are automatically unwrapped in templates, so `:aspect-ration="aspectRatio"` is enough